man/systemd.network.xml

```diff
@@ -706,6 +706,12 @@
             <citerefentry project='man-pages'><refentrytitle>inet_pton</refentrytitle><manvolnum>3</manvolnum></citerefentry>.</para>
           </listitem>
         </varlistentry>
+        <varlistentry>
+          <term><varname>OnDevice=</varname></term>
+          <listitem>
+            <para>A boolean. Specifies whether to take device as default route.</para>
```

Comment:
But I wonder what the on-device stuff precisely means then? we probably should grok what is going on first, before we add this option! can you figure out what the precise effect of the on-device stuff is?